bakerydemo/templates/base/home_page.html

```diff
@@ -26,38 +26,29 @@ <h1>{{ page.title }}</h1>
 
 <div class="container">
     <div class="row promo-row">
-        <div class="col-sm-5 promo">
-            {% if page.promo_image or page.promo_title or page.promo_text %}
-                <figure>{% image page.promo_image fill-200x200-c100 %}</figure>
-                <div class="promo-text">
-                    <h3>{{ page.promo_title }}</h3>
-                    {{ page.promo_text|safe }}
-                </div>
+        <div class="featured-cards col-sm-5 col-sm-offset-1 ">
+            {% if page.featured_section_1 %}
+                <h2 class="featured-cards__title">{{ page.featured_section_1_title }}</h2>
+                <ul class="featured-cards__list">
+                    {% for childpage in page.featured_section_1.specific.children|slice:"3" %}
+                        {% include "includes/card/listing-card.html" with title=childpage.title card_image=childpage.image %}
+                    {% endfor %}
+                </ul>
+                <a class="featured-cards__link" href="/breads">
+                    <span>View more of our breads</span>
+                    {% include "includes/chevron-icon.html" with class="featured-cards__chevron-icon" %}
+                </a>
             {% endif %}
         </div>
 
-        <div class="col-sm-6 col-sm-offset-1 feature-1">
-            {% if page.featured_section_1 %}
-            <h2>{{ page.featured_section_1_title }}</h2>
-                <div class="featured-children">
-                    {% for childpage in page.featured_section_1.specific.children|slice:"4" %}
-                        <li>
-                            <div class="row">
-                                <div class="col-xs-4">
-                                    <a href="{{childpage.url}}">
-                                        <figure>
-                                            {% image childpage.image fill-180x140-c100 %}
-                                        </figure>
-                                    </a>
-                                </div>
-                                <div class="col-xs-8">
-                                    <h3><a href="{{childpage.url}}">{{childpage.title}}</a></h3>
-                                </div>
-                            </div>
-                        </li>
-                    {% endfor %}
+        <div class="col-sm-6 promo">
+            {% if page.promo_image or page.promo_title or page.promo_text %}
+                <div class="col-lg-10 promo-text">
+                    <h2>{{ page.promo_title }}</h2>
+                    {{ page.promo_text|safe }}
                 </div>
             {% endif %}
+            <figure>{% image page.promo_image fill-590x413-c100 %}</figure>
```

Comment:
question: should any of these 3 be wrapped in a conditional? i.e. can you add a title and image but no text?

Reply:
All of these are optional, I will add the conditional statements in case the content is forgotten